### PR TITLE
[node] Put newest versions of files in the root

### PR DESCRIPTION
**Pull request by jablko**
If there's only one version of a file, shared by all `typesVersions`, put it in the root.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

**Comment by SimonSchick** [comment]
This makes sense and reduced future diff sizes.

**Review by SimonSchick — APPROVED** [review]

**Comment by Flarna** [comment]
fyi @sanders - seems something similar was done by you in #47195

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -15,7 +15,10 @@ jobs:
       - run: npm install
 
       # Run tests
-      - run: npm run test-all
+      - run: |
+          echo adone >> node_modules/@definitelytyped/dtslint-runner/expectedFailures.txt
```
Comment (sandersn) [inline .github/workflows/CI.yml:19]: given that that's just a 4.1-failing dependent of node, can you revert this change? we can merge this despite the failure.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -15,7 +15,10 @@ jobs:
...
+          sed --in-place 's%return undefined;%//&%' node_modules/@definitelytyped/dtslint-runner/dist/main.js
```
Comment (sandersn) [inline .github/workflows/CI.yml:20]: can you make this change to @definitelytyped/dtslint-runner instead of patching it here?

**Review by sandersn — CHANGES_REQUESTED** [review]
This is a really good idea. Can you revert the changes to CI.yml/azure-pipelines.yml?

File: .github/workflows/CI.yml (hunk 1 above)
Comment (jablko) [reply .github/workflows/CI.yml]: :+1: Thanks, done.

File: .github/workflows/CI.yml (hunk 2 above)
Comment (jablko) [reply .github/workflows/CI.yml]: Will do!

**Review by sandersn — APPROVED** [review]

**Comment by sandersn** [comment]
The build failure is from `adone` which is expected; I'll try to fix it in the next day or two. Let's merge this PR.

**Comment by panva** [comment]
> The build failure is from `adone` which is expected; I'll try to fix it in the next day or two. Let's merge this PR.

@sandersn other PRs to node are also blocked by `adone`. Not to mention they all need rebase now...

**Pull request merged**